client/src/styles/new-song-form.css

```diff
@@ -0,0 +1,71 @@
+.new-song-form {
+    width: 50%;
+    margin: 2% auto 0;
+    background: white;
+    border: 2px solid #e6e6e6;
+    border-radius: 4px;
+}
+
+.song-link-input {
+    margin-top: 3%;
+    height: 20px;
+    width: 80%;
+    background: #fafafa;
+}
+
+.select-format {
+    width: 80%;
+    margin-top: 3%;
+    height: 25px;
+    border: 1px solid #e6e6e6;
+    border-radius: 4px;
+    font-size: 18px;
+    transition: background-color 3s ease;
+    background: #fafafa;
+}
+
+.add-song-button {
+    font-size: 18px;
+    margin-top: 4%;
+    margin-bottom: 2%;
+}
+
+.song-form-button:hover {
+    background: #222222;
+}
+
+.loader {
+    margin: 2% auto;
+    border: 16px solid #f3f3f3; /* Light grey */
+    border-top: 16px solid #222222; /* Blue */
+    border-radius: 50%;
+    width: 130px;
+    height: 130px;
+    animation: spin 1.3s linear infinite;
+}
+
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+
+.download-song-info-error-form {
+    text-align: center;
+    padding: 2%;
+    font-weight: bold;
+    font-size: 17px;
+}
+
+.song-form-button {
+    cursor: pointer;
+    color: beige;
+    border-radius: 10px;
+    background: #333333;
+    height: 35px;
+    width: 160px;
+}
+.ok-button {
```

Comment:
If there is **only** one `ok-buttton` than it should have an id, not a class.
I know we said in the course that you shouldn't use `id` at all - but it's not true. They can help in making your app faster